### ISSUE TITLE
How do enable multi GPU training?

### ISSUE THREAD

**Issue by AmrutaMuthal** [bourdakos1/Custom-Object-Detection#37]

**Comment by AmrutaMuthal** [comment]
I am trying to use this on a multi GPU cloud system. Not sure which parameters to change for utilizing all the cores

**Comment by AmrutaMuthal** [comment]
I am running this on

**Comment by bourdakos1** [comment]
Try `num_clones=YOUR_GPU_NUMS`

**Comment by bourdakos1** [comment]
Hi @AmrutaMuthal, this repo relies on a dated version of the TensorFlow object detection api. We've moved to a more future proof version here: https://github.com/cloud-annotations/training

I think you rely on train.py for training on multiple gpus, however this has been removed in the new version (no multi-gpu support in the latest version (its coming when the switch to keras)). In the meantime you can set the revision flag to a version that includes the train.py

**Comment by AmrutaMuthal** [comment]
I was able to train using all the GPUs with the num_clones option. I ended up with a low volatile loss. I expected training speed to increase by that didn't happen either. Realised that data needs some cleanup. I have some very blurred and dark images in my training set. Removing those should help with the loss at least. Not sure how to improve training speed. I am getting close to 1.2 Sec/epoch with a training set of 10K images and batch size =1
